core/src/main/java/org/infinispan/globalstate/GlobalConfigurationManager.java

```diff
@@ -26,6 +26,24 @@
     */
    Cache<ScopedState, Object> getStateCache();
 
+   /**
+    * Defines a cluster-wide configuration template
```

Comment:
In keeping with my previous comment, `Defines a cache template cluster-wide` would sound more friendly to LOCAL caches. It would also fit better with @oraNod's in-progress terminology PR (#8742).